plugins/Toolbox/src/Toolbox.py

```diff
@@ -661,6 +679,27 @@ def _onRequestFinished(self, reply: QNetworkReply) -> None:
                                 # Tell the package manager that there's a new set of updates available.
                                 packages = set([pkg["package_id"] for pkg in self._server_response_data[response_type]])
                                 self._package_manager.setPackagesWithUpdate(packages)
+                            elif response_type == "installed_packages":
+                                user_subscribed = [(plugin['package_id'], plugin['package_version']) for plugin in json_data['data']]
+                                Logger.log("i", "- User is subscribed to {} package(s).".format(len(user_subscribed)))
```

Comment:
Why add the "- " in front of the message (and also, should it be info? it feels more like a debug message)

Reply:
Good catch, must have slipped among some print statements from previous.